ilc/client/BundleLoader.js

```diff
@@ -40,21 +40,36 @@ export class BundleLoader {
         const waitTill = [this.loadApp(appName)];
 
         if (app.cssBundle) {
-            waitTill.push(this.loadCss(app.cssBundle));
+            waitTill.push(this.loadCss(app.cssBundle, appName));
         }
 
         return Promise.all(waitTill).then(values => values[0]);
     }
 
-    loadCss(url) {
-        return this.#systemJs.import(url).catch(err => { //TODO: inserted <link> tags should have "data-fragment-id" attr. Same as Tailor now does
+    loadCss(url, appName) {
+        if (!url) {
+            return Promise.resolve();
+        }
+
+        return this.#systemJs.import(url)
+        .then(() => {
+            const currentLink = document.querySelector(`link[href="${url}"]`)
+            currentLink.setAttribute('data-fragment-id', appName)
```

Comment:
Minor: To me it looks like something that should have been done at SSR/inside System.js.